src/SFA.DAS.Reservations.NServiceBusListener/CreatedEventHandler.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using SFA.DAS.Reservations.Messages;
+
+namespace SFA.DAS.Reservations.NServiceBusListener
+{
+    public class CreatedEventHandler : IHandleMessages<ReservationCreatedEvent>
+    {
+        public Task Handle(ReservationCreatedEvent message, IMessageHandlerContext context)
+        {
+            Console.WriteLine($"Event recieved: {message.Id}{message.StartDate}");
```

Comment:
Can you quickly format the message in a clearer structure? I was meant to change it but forgot to at the time.

Reply:
Done